src/backend/API/members.js

```diff
@@ -40,4 +40,21 @@ router.get("/:id", async (request, response) => {
   }
 });
 
+
+router.delete("/", async (request, response) => {
+  const { id } = request.body;
+  try {
+    const deletedMember = await db("members").where({ id }).del();
+    if (deletedMember) {

```

Comment:
Your implementation looks good! Personally, I'm not a big fan of if-else structures, so I might suggest an alternative like this for better readability. However, it's perfectly fine the way you've written it too:
 ```js
 if (!deletedMember) {
  return response.status(404).json({ error: "Member not found" });
}
response.json({ message: "Member deleted successfully" });

 ```